apps/clinical/src/components/forms/medicationRequest/styles/SelectedMedicationRequestItem.module.scss

```
@@ -7,11 +7,11 @@
   margin-left: 0;
 }
 
-.medicationTitle {
+.itemTitle {
   @include type-style("heading-compact-02");
   margin-bottom: $spacing-04;
```

Stylelint (17.11.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)